[13.0][FIX] l10n_es_aeat_sii_oca: SII Refund Type not required

joao-p-marques opened the pull request:
Make the SII Refund Type field not required nor available for `account.move` records that are not `in_invoices` or `out_invoices`

@Tecnativa
TT30339

ping @pedrobaeza @chienandalu

pedrobaeza commented on l10n_es_aeat_sii_oca/wizards/account_move_reversal.py:
```diff
@@ -9,8 +9,12 @@ class AccountMoveReversal(models.TransientModel):
+        invoices = self.env["account.move"].browse(self.env.context.get("active_ids"))
+        # If any of the invoices part of the active_ids match the criteria,
+        # show the field
+        return any(invoices.mapped("company_id.sii_enabled")) and invoices.filtered(
+            lambda i: i.move_type in ("in_invoice", "out_invoice")
+        )
```
Please simplify the expression and in v13, it's `type` AFAIK:
```suggestion
        return bool(invoices.filtered(
            lambda i: i.type in ("in_invoice", "out_invoice") and i.company_id.sii_enabled
        )
```

joao-p-marques replied:
> in v13, it's type AFAIK

Sorry, MB.

Changes made :+1:

joao-p-marques committed:
[FIX] l10n_es_aeat_sii_oca: SII Refund Type not required

Make the SII Refund Type not required nor available for account.move records that are not "in_invoices" or "out_invoices"

TT30339

pedrobaeza commented:
/ocabot merge patch

Please fw-port it to v14
